src/Cache/RedisCacheTagsChecksum.php

```diff
@@ -24,7 +24,7 @@ class RedisCacheTagsChecksum implements CacheTagsChecksumInterface, CacheTagsInv
    *
    * @var array
    */
-  protected $tagCache = array();
+  protected $tagCache = [];
```

Comment:
Sorry for the noise here, just found this to be more readable, even though I don't really care about writing it back as "array()".